src/editor/nodes/KitPieceNode.js

```diff
@@ -38,7 +38,7 @@ export default class KitPieceNode extends EditorNodeMixin(Model) {
             node.activeClipIndex = loopAnimationComponent.props.activeClipIndex;
           } else if (clip !== undefined && node.model && node.model.animations) {
             // DEPRECATED: Old loop-animation component stored the clip name rather than the clip index
-            node.activeClipIndex = node.model.animations.findIndex(animation => animation.name === clip);
+            node.activeClipIndex = [node.model.animations.findIndex(animation => animation.name === clip)];
```

Comment:
Can we change this to `activeClipIndices`?

Reply:
Updated.

Note that we still need to support the previous struct for backwards compatibility.